include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -623,7 +623,8 @@ typedef struct OrtOpenVINOProviderOptions {
                                  cache_dir{},
                                  context{},
                                  enable_opencl_throttling{},
-                                 enable_dynamic_shapes{} {}
+                                 enable_dynamic_shapes{},
+                                 queue{} {}
```

Comment:
The struct is frozen and its our legacy API. Kindly upgrade to ORT ProviderOptions Map structure